CLAUDE.md

```
@@ -0,0 +1,393 @@
+    invalid_citation = "Brown v Board of Education"
+    assert validate_bluebook_citation(invalid_citation) is False
+```
+
+### Git Workflow
+- **Branch Naming**:
+  - `feature/description` for new features
+  - `fix/description` for bug fixes
+  - `refactor/description` for refactoring
+  - `docs/description` for documentation
+- **Commit Messages**: Conventional Commits format
+  ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)